sympy/printing/octave.py

```diff
@@ -432,6 +435,23 @@ def _print_LambertW(self, expr):
         return "lambertw(" + args + ")"
 
 
+    def _print_math_func(self, expr):
+        known = self.known_functions[expr.__class__.__name__]
+        args = self._print(expr.args[0])
+        args += ', %s' % self._print(expr.func(*expr.args[1:]))
+
+        return '{name}({args})'.format(
+            name=known,
+            args=args
```

Comment:
same for args, just use `', '.join(map(self._print, expr.args))`

Reply:
sorry my bad, that does not work here (due to nesting)

Reply:
It could perhaps be wise to rename this method to something along the line of `_nested_binary_math_func` just to signal that.